doc/htmldoc/ref_material/pynest_api/index.rst

```diff
@@ -0,0 +1,358 @@
+      {% endfor %}
+
+
+:doc:`Functions related to randomizaton  <nest.random.hl_api_random>`
+---------------------------------------------------------------------
+
+.. automodule::   nest.random.hl_api_random
+   :noindex:
+
+
+   .. autosummary::
+
+      {% for key, value in api_dict.items() %}
+          {% if key == 'hl_api_random' %}
+              {% for item in value %}
+                  {{ item }}
+              {% endfor %}
+          {% endif %}
+      {% endfor %}
+
+
+:doc:`Functions related to parallel computing  <nest.lib.hl_api_parallel_computing>`
+------------------------------------------------------------------------------------
+
+.. automodule::   nest.lib.hl_api_parallel_computing
+   :noindex:
+
+
+   .. autosummary::
+
+      {% for key, value in api_dict.items() %}
+          {% if key == 'hl_api_parallel_computing' %}
+              {% for item in value %}
+                  {{ item }}
+              {% endfor %}
+          {% endif %}
+      {% endfor %}
+
+
+:doc:`Functions related NEST server  <nest.server.hl_api_server>`
+-----------------------------------------------------------------
+
+.. automodule::   nest.server.hl_api_server
+   :noindex:
+
+
+   .. autosummary::
+
+      {% for key, value in api_dict.items() %}
+          {% if key == 'hl_api_server' %}
+              {% for item in value %}
+                  {{ item }}
+              {% endfor %}
+          {% endif %}
+      {% endfor %}
+
+
+:doc:`Functions related to voltage trace  <nest.voltage_trace>`
+----------------------------------------------------------------
+
+.. automodule::   nest.voltage_trace
+   :noindex:
+
+
+   .. autosummary::
+
+      {% for key, value in api_dict.items() %}
+          {% if key == 'voltage_trace' %}
+              {% for item in value %}
+                  {{ item }}
+              {% endfor %}
+          {% endif %}
+      {% endfor %}
+
+
+:doc:`Functions related to raster plots <nest.raster_plot>`
+------------------------------------------------------------
+
+.. automodule::   nest.raster_plot
+   :noindex:
+
+
+   .. autosummary::
+
+      {% for key, value in api_dict.items() %}
+          {% if key == 'raster_plot' %}
+              {% for item in value %}
+                  {{ item }}
+              {% endfor %}
+          {% endif %}
+      {% endfor %}
+
+
+:doc:`Functions related to plotting networks  <nest.visualization>`
+-------------------------------------------------------------------
+
+.. automodule::   nest.visualization
+   :noindex:
+
+
+   .. autosummary::
+
+      {% for key, value in api_dict.items() %}
+          {% if key == 'visualization' %}
+              {% for item in value %}
+                  {{ item }}
+              {% endfor %}
+          {% endif %}
+      {% endfor %}
+
+
+:doc:`Functions related to kernel attributes  <kernel_attributes>`
+------------------------------------------------------------------
+
+.. autoclass:: nest.NestModule
+   :noindex:
```

Comment:
Ideally, I'd like to see the attributes on the same level as the functions in the other sections, without the `class NestModule(name)`  header. Obviously ignore this request for change if implementing it would mean that all of them would have to be listed individually or something like that.

![Screenshot from 2023-08-15 11-00-41](https://github.com/nest/nest-simulator/assets/2678428/06d09d21-2f03-4623-a493-d452357f8c00)

Reply:
NestModule line removed, indentation removed